packages/stacks-classic/lib/components/button/button.less

```diff
@@ -214,29 +209,46 @@
             margin-top: -0.3em; // [4]
             margin-bottom: -0.3em; // [4]
             transition: opacity 200ms var(--te-smooth); // Animate the transition to .is-loading
+            
+            &:not(:last-child) {
+                margin-right: var(--_bu-icon-gap, 8px);
```

Comment:
No need for the fallback `8px` value here. Since `--_bu-icon-gap` is defined in a broader scope, it should always have a value.